admin/includes/init_includes/init_aidba_install.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+if (!defined('IS_ADMIN_FLAG')) {
+    die('Illegal Access');
+}
+if(!defined('ADDITIONAL_IMAGES_APPROACH')) {
+    $db->Execute("INSERT INTO " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) VALUES ('Additional Images Approach', 'ADDITIONAL_IMAGES_APPROACH', 'old', 'Enable Additional Images Module', '4', '100', 'zen_cfg_select_option(array(\'old\', \'modern\'), ', NOW())");
+}
+
+if(!$sniffer->table_exists(TABLE_PRODUCTS_ADDITIONAL_IMAGES) && defined('ADDITIONAL_IMAGES_APPROACH') && ADDITIONAL_IMAGES_APPROACH == 'modern') {
+    // alter product table to InnoDB if not already done
+    $db->Execute("ALTER TABLE " . TABLE_PRODUCTS ." ENGINE=InnoDB;");
+
+    // create products_additional_images table
+    $db->Execute("CREATE TABLE " . TABLE_PRODUCTS_ADDITIONAL_IMAGES . " (
+            id INT(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+            products_id INT(11) NOT NULL,
+            additional_image VARCHAR(255) NOT NULL,
+            sort_order INT(11) DEFAULT 0,
+            created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+            FOREIGN KEY (products_id) REFERENCES " . TABLE_PRODUCTS ."(products_id) ON DELETE CASCADE
```

Comment:
Should there be a space between the table name and the `(products_id) ON DELETE CASCADE`?

Reply:
No, I don't think so. According to MySQL [docs](https://dev.mysql.com/doc/refman/8.4/en/create-table-foreign-keys.html) and the example provided, there's no space between the table name and field name that it references to:
`CREATE TABLE parent (
    id INT NOT NULL,
    PRIMARY KEY (id)
) ENGINE=INNODB;

CREATE TABLE child (
    id INT,
    parent_id INT,
    INDEX par_ind (parent_id),
    FOREIGN KEY (parent_id)
        REFERENCES parent(id)
        ON DELETE CASCADE
) ENGINE=INNODB;`